compiler/installer.ini

```diff
@@ -6,7 +6,7 @@ Name: "Nim"
 Version: "$version"
 Platforms: """
   windows: i386;amd64
-  linux: i386;hppa;ia64;alpha;amd64;powerpc64;arm;sparc;sparc64;m68k;mips;mipsel;mips64;mips64el;powerpc;powerpc64el;arm64;riscv32;riscv64
+  linux: i386;hppa;ia64;alpha;amd64;powerpc64;arm;sparc;sparc64;m68k;mips;mipsel;mips64;mips64el;powerpc;powerpc64el;arm64;riscv32;riscv64;loongarch64
```

Comment:
Not familiar with this architecture, but perhaps we shouldn't be generating official C sources for it (if it's very rare). But I dunno, maybe C source bloat isn't too big of a concern right now.

Reply:
@dom96 yeah, I think you're right, Araq did remove e2k from installer.ini in my PR so I'm assuming it's to not make the csources too big.

Reply:
```suggestion
  linux: i386;hppa;ia64;alpha;amd64;powerpc64;arm;sparc;sparc64;m68k;mips;mipsel;mips64;mips64el;powerpc;powerpc64el;arm64;riscv32;riscv64
```